packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -565,77 +317,18 @@ export class RecentsWidget extends Component {
     props.updateSpaceKeywordFilter(keyword);
   }
 
-  /**
```

Comment:
None of this is needed anymore with the new redesign since they do not have activity previews on the space list.